packages/rn-tester/Podfile

```
@@ -66,6 +68,7 @@ target 'RNTesterIntegrationTests' do
 end
 
 post_install do |installer|
-  react_native_post_install(installer, @prefix_path)
+  mac_catalyst_enabled = false
+  react_native_post_install(installer, mac_catalyst_enabled, @prefix_path)
```

Instead of defining a variable, which I think is done just to have the name of the parameter for a code-as-documentation purpose, why don't we use named parameters from Ruby?

We can define a function with this syntax:
```ruby
def foo(named_param: false) 
end
```

and then invoke the function with this syntax:

```ruby
foo(:named_param => true)
```

with this, we can emove the variable declaration and update this code as it follows:

```suggestion
react_native_post_install(installer, @prefix_path, :mac_catalyst_enabled => false)
```